skimage/restoration/_nl_means_denoising.pyx

```diff
@@ -353,7 +362,7 @@ def _nl_means_denoising_3d(image, int s=7, int d=13, double h=0.1,
                                     padded[pln_start_i:pln_end_i,
                                            row_start_j:row_end_j,
                                            col_start_k:col_end_k],
-                                    w, s, var*var)
+                                    w, s, var)
```

Comment:
The change at this line is a bug fix to my previously merged PR #2890  (I was originally squaring the standard deviation here, but later switched to using the variance and missed removing the square in this case)